### DISCUSSION TITLE
Visual change for mods that have difficulty ajust

### DISCUSSION THREAD

**Discussion by Biokax** [ppy/osu#16129] [Ideas] [answered]
Now osu have a lot of mods and some has difficulty adjust options.
Currently we can't differentiate for the same mod if a player has activated a difficulty adjust option or not.
For example HD mod with the only fade approach circles reduce the difficulty of the mod, but on the leaderboard and on the screen result we can't see if it has been used or not.

The proposal is to change the color of the mod:

- blue or green if the option reduces the difficulty of the mod
- normal for no option selected
- orange or red for option that increase the difficulty of the mod

I know that it have a conflict with default mod color, but that for the idea

I also think about an acronym refactor to come with that.
Something like "+/-"
For HD with only fade approach circles selected it become "HD-" or something like that.

This system is far from being perfect but I really think that players need to know if a mod have options selected or not without having to watch the replay.

**Comment by NotGumballer91** [comment]
You don't have to watch the replay. You can still see the mod's customization by hovering over it.

![image](https://user-images.githubusercontent.com/64581009/146549338-56c73143-5eb2-41e9-bc3d-950c9b2119a0.png)

**Reply by Biokax** [reply]
right, but without hovering it you can't see that its different. For example when scores will come to the website you will need to hover each scores to know if its the default mod or it have mod customization ?

When we talk about HD with customization does we call it HD or HD- ? because its way more simple that HD and that still called HD

**Reply by NotGumballer91** [reply]
I guess the easier solution would be to just add an overlay on it. To simply tell it's "Modified".

**Comment by smoogipoo** [comment]
I've brought this up with @arflyte, and I think we'll have some place on the results screen listing all mod settings (perhaps in the click-through statistics).

**Comment by peppy** [comment, marked as answer]
Definitely agree with a modified light/circle/something somewhere on the mod icon.

And potentially a single number "bubble" or "pill" display for cases like speed-adjust mods, where having it immediately visible would be preferred.

**Reply by Moltanicaa** [reply]
A blue DT icon at a 1.1x multiplier (for example) and a red icon at 1.75x would be pretty nice. I'd like to see such work. Although this will make the mod section look like a rainbow

**Reply by bdach** [reply]
The concept of "mod multipliers" isn't really guaranteed to live for much longer, from what I recall.

**Reply by Moltanicaa** [reply]
Did we understand each other correctly? I mean, the classic DT speed up the map by 1.5x, and with customization, you can from 1.01x to 2x.
